packages/zosjobs/__tests__/__system__/SubmitJobs.system.test.ts

```diff
@@ -32,25 +30,16 @@ const badUSSFile = "/tmp/does/not/exist/fake.txt";
 
 const LONG_TIMEOUT = 100000; // 100 second timeout - jobs could take a while to complete due to system load
 
-const waitThreeSeconds = () => {
-    return new Promise<void>((resolveWaitTime) => {
-        const threeSeconds = 3000;
-        setTimeout(() => {
-            resolveWaitTime();
-        }, threeSeconds);
-    }
-    );
-};
+const wait = promisify(setTimeout);
```

Comment:
Big brain! 🧠 
I'm definitely using this for writing one-time scripts to test stuff 😋